dataverse-webapp/src/main/webapp/resources/vecler/theme.scss

```diff
@@ -1205,6 +1205,19 @@ tr.ui-state-highlight a:not(.btn), .ui-widget-content tr.ui-state-highlight a:no
 	width: 8em;
 }
 
+/* Disable selecting single table row accidentally */
+.ui-datatable-selectable td {
+	cursor: default;
```

Comment:
there is a weird effect when you click on row (where before there was action to select checkbox). The whole row is blinking. 
I think that we should do something with `<p:dataTable>` in xhtml's. There is a param `rowSelectMode="checkbox"` and it causes the same effect (selecting only via checkbox). But I would keep:
```
.ui-datatable-selectable td {
	cursor: default;
}
```
because for some reason with changed rowSelectMode cursor on the whole row is hand not classic pointer

Reply:
Thanks for the tip, I've made appropriate changes where necessary